pkg/apiserver/openapi/zz_generated.openapi.go

```diff
@@ -4855,7 +4855,7 @@ func schema_k8sio_api_core_v1_ContainerImage(ref common.ReferenceCallback) commo
 				Properties: map[string]spec.Schema{
 					"names": {
 						SchemaProps: spec.SchemaProps{
-							Description: "Names by which this image is known. e.g. [\"k8s.gcr.io/hyperkube:v1.0.7\", \"dockerhub.io/google_containers/hyperkube:v1.0.7\"]",
+							Description: "Names by which this image is known. e.g. [\"registry.k8s.io/hyperkube:v1.0.7\", \"dockerhub.io/google_containers/hyperkube:v1.0.7\"]",
```

Comment:
The Github workflow that checks that all auto-generated files are up-to-date is failing for this PR. You need to revert the change to this file.

This file is auto-generated so making a manual change to it (I assume you ran a `sed` command when working on this PR) is typically incorrect. When we update our K8s dependencies in the future, the reference to `k8s.gcr.io` will go away, but in the mean time, having it around is not an issue.